grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/syntaxtree/utils/CallerUtils.java

```diff
@@ -151,6 +151,21 @@ public static Class getCallerClass(String key, String value) {
         complete.addQualifiers(new String[]{"isolated", "remote"});
         caller.addMember(complete.getFunctionDefinitionNode());
 
+        Function isCancelled = new Function("isCancelled");
+        isCancelled.addReturns(NodeFactory.createBuiltinSimpleNameReferenceNode(
+                SyntaxKind.BOOLEAN_TYPE_DESC,
+                AbstractNodeFactory.createIdentifierToken("boolean")
+        ));
```

Comment:
Shall we use the `getBuiltinSimpleNameReferenceNode()` function in `TypeDescriptor` class (https://github.com/ballerina-platform/module-ballerina-grpc/blob/5c8ea6868f834cc0b3b9bd81515923fe3e0f645c/grpc-native/src/main/java/org/ballerinalang/net/grpc/builder/syntaxtree/components/TypeDescriptor.java#L97)
eg:
```
isCancelled.addReturns(TypeDescriptor.BuiltinSimpleNameReferenceNode("boolean"))
```
It would be easier to maintain if we keep `NodeFactory` related functions in `components` package and `grpc` module-specific util functions in `utils` package.

Best if we could keep `grpc` module-specific util functions in `utils` package and `NodeFactory` related functions in `components` package.

Reply:
fixed in [#934a43c](https://github.com/ballerina-platform/module-ballerina-grpc/pull/293/commits/934a43ccea71780fd6dae4c71a74ab9b22ef6d9e)